src/coreclr/jit/emitarm64.cpp

```diff
@@ -29310,6 +29328,23 @@ void emitter::emitDispInsHelp(
             emitDispFloatImm(emitGetInsSC(id));                 // iiiiiiii
             break;
 
+        // DUP <Zd>.<T>, #<imm>{, <shift>}
+        // MOV <Zd>.<T>, #<imm>{, <shift>}
+        case IF_SVE_EB_1A: // ........xx...... ..hiiiiiiiiddddd -- SVE broadcast integer immediate (unpredicated)
+        {
+            imm                 = emitGetInsSC(id);
```

Comment:
How about moving into a helper?
`imm = ExtractSimm8_MultipleOf256()`

Reply:
If we encode `imm` in `emitIns`, this logic goes away.

Reply:
good we don't have to check here if it was small or large constant. The `emitGetInsSC()` does it for us.